### ISSUE TITLE
Undefined CellType in river-deadend-narrow.track

### ISSUE THREAD

**Issue by GMMDMDIDEMS** [koehlma/momba#11]
In the file `river-deadend-narrow.track` an undefined CellType `i` is used:

> dim: 28 63
xxxxxgggggxxxxxxxxgggggxxxxxxxxxgggggxxxgggggxxxxxxxxgggggxxxxx
xxxxx.....xxxxxxxx.....xxxxxxxxx.....xxx.....xxxxxxxx.....xxxxx
xxxxx.....xxxxxxxx.....xxxxxxxxx.....xxx.....xxxxxxxx.....xxxxx
xxxxx.....xxxxxxx.......xxxxxxxx.....xx.......xxxxxxx.....xxxxx
xxxxxx.....xxxx.........xxxxxxxxx.....x.........xxxx.....xxxxxx
xxxxxxx...........xxxxxxxxxxxxxxx..........xx...........xxxxxxx
xxxxxxxxx......xxxxxxxiiiiixxxxxxxxx.....xxxxxxx......xxxxxxxxx
xxxxxxxxxx....xxxxxxxxiiiixxxxxxxxxxx....xxxxxxxx....xxxxxxxxxx
xxxxxxxxxx....xxxxxxxxiiiixxxxxxxxxxx....xxxxxxxx....xxxxxxxxxx
xxxxxxxxxx.....xxxxxxiiiiiiiiixxx.........xxxxxx.....xxxxxxxxxx
xxxxxxxxxx.....xxxxiiiixxxiiiiix.....xxx....xxxx.....xxxxxxxxxx
xxxxxxxxxxxx...........xxx...........xxx...........xxxxxxxxxxxx
xxxxxxxxxxxxx........xxxxxxx.......xxxxxxx........xxxxxxxxxxxxx
xxxxxxxxxxxxxx....xxxxxxxxxxx.....xxxxxxxxxxx....xxxxxxxxxxxxxx
xxxxxxxxxxxxxx....xxxxxxxxxx.......xxxxxxxxxx....xxxxxxxxxxxxxx
xxxxxxxxxxxxxx....xxxxxxxx...........xxxxxxxx....xxxxxxxxxxxxxx
xxxxxxxxxxxxxxx.......xx......xxx......xx.......xxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxx..........xxxxxxx..........xxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxx......xxxxxxxxxxx......xxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxx.....xxxxxxxxxxx.....xxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxx.....xxxxxxxxxxx.....xxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxiiiiixxxxxxxiiiiixxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxiiiiixxxiiiiixxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxiiiiiiiiixxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxiiiiiiixxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxiiiiixxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxiiiiixxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxsssssxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Should `i` be replaced by `.`?

**Comment by koehlma** [comment]
To be honest, I am not sure what the `i` stands for. I will check with the person who created them and then come back to you.

**Comment by koehlma** [comment]
Yes, the `i` marks “interesting regions”. They can be replaced with `.`.
